### DISCUSSION TITLE
Does the Mac version have startup on boot and minimize to the tray functions? Currently, when I close the window, the remote session ends.

### DISCUSSION THREAD

**Discussion by zxjzxj0010** [rustdesk/rustdesk#9583] [Q&A]
请问mac版本有开机自启动和最小化到托盘的功能吗，现在关闭窗口，远程就结束了Does the Mac version have startup on boot and minimize to the tray functions? Currently, when I close the window, the remote session ends.

**Comment by rustdesk** [comment]
is it hard to see this?

![image](https://github.com/user-attachments/assets/f545d519-8333-40de-8bf1-eec32ac88b8a)

**Reply by zxjzxj0010** [reply]
It wasn't installed in the default location originally,and i can not see this tip, thank you for help.

**Reply by heavy008** [reply]
我的也是，更新后消失了，卸载重装就可以了

**Reply by qjzy** [reply]
原来如此！我之前将软件放在了外置硬盘里，所以没有看到安装提示。现在把软件移到(Application) 文件夹后，安装提示就正常出现了，问题已解决。感谢您的指导！
Got it! I previously placed the software on an external drive, which caused the installation prompt to be missed. After moving it to the "Applications" folder, the prompt appeared as expected, and the issue is resolved now. Thanks for your help!
